filebeat/tests/system/test_registrar.py

```diff
@@ -851,6 +851,9 @@ def test_clean_removed(self):
         """
         Checks that files which were removed, the state is removed
         """
+
+        raise SkipTest
```

Comment:
Add a comment mentioning the reason why the test is skipped? or with a link to the issue?